scripts/avd_magisk.sh

```diff
@@ -66,7 +66,7 @@ fi
 
 # Stop zygote (and previous setup if exists)
 magisk --stop 2>/dev/null
-stop zygote
+stop
```

Comment:
This also stop adbd, will terminate script execution in interactive shell .

Reply:
The test will not properly execute if only using `stop zygote`